chartlets.js/src/lib/components/Plot.tsx

```diff
@@ -2,32 +2,109 @@ import { VegaLite } from "react-vega";
 
 import { type PlotState } from "@/lib/types/state/component";
 import { type ComponentChangeHandler } from "@/lib/types/state/event";
+import {
+  isString,
```

Comment:
Move into `src/lib/utils/isString.ts` and use in other 3 places in chartlets code.

Reply:
Which 3 other places? It is only being used in one place

Reply:
Ah okay,I misunderstood that this function is being used 3 other places in the Plot component.